test/ng/filter/orderBySpec.js

```diff
@@ -8,6 +8,11 @@ describe('Filter: orderBy', function() {
 
 
   describe('(Arrays)', function() {
+    it('should throw an exception if no array-like object is provided', function() {
+      expect(function() { orderBy({}); }).
+        toThrowMinErr('orderBy', 'notarray', 'Expected array but received: {}');
+    });
+
```

Comment:
Please add a test that shows that an array-like (i.e. not a pure array) does not fail, unless there is already such a test in the spec.